Configuration/Settings.yaml

```diff
@@ -12,6 +13,8 @@ FormatD:
     api:
       portalId: ''
       accessToken: ''
+    fileApi:
+      folderId: ''
```

Comment:
maybe better naming would be `defaultFolderId` as it is only used as fallback is no folterId is provided. Also i would suggest having only one `api` key in the config and just adding a new key. So `FormatD.HubSpot.api.defaultUploadFolderId`. As i understand it the accessToken ist the same.

Reply:
Done ✅